modules/claims_api/lib/custom_error.rb

```diff
@@ -29,7 +29,7 @@ def build_error
 
     private
 
-    def raise_backend_exception(_key = 'EVSS')
+    def raise_backend_exception
```

Comment:
There are still arguments being passed in in the calls to this method above.  Lines 17, 21, 24.  If those do not get removed then it wil raise an argument error when this gets used.